### PR TITLE
Update meta info for twitter card

### PR DESCRIPTION
**Pull request by NotJohs**
PR to fix [#2206](https://github.com/opencollective/opencollective/issues/2206)

### CODE REVIEW

**Comment by znarf** [comment]
@xdamman here is the result.

https://cards-dev.twitter.com/validator
https://opencollective-frontend-klzaywpbri.now.sh/babel

Capture d’écran 2019-07-06 à 20 45 32

Are you ok with that? Any feedback?

**Comment by NotJohs** [comment]
@xdamman Looks great

**Comment by xdamman** [comment]
This PR introduced this bug:

**Comment by xdamman** [comment]
Sharing the homepage on Twitter now looks like this which is a regression



It's only when `title` is defined that we should change the title of the page.

File: src/components/Header.js [hunk 1]
```diff
@@ -20,13 +20,17 @@ class Header extends React.Component {
...
-    const { description, image, twitterHandle } = props;
+    const { description, image, twitterHandle, title } = props;
     const meta = {
       'twitter:site': 'opencollect',
       'twitter:creator': twitterHandle,
       'fb:app_id': '266835577107099',
       'og:image': image,
       description: truncate(description, 256),
+      'twitter:card': 'summary',
+      'twitter:title': `${title} - Open Collective`,
```
Comment (xdamman) [inline src/components/Header.js]: It's only when `title` is defined that we should add it to that string.

```js
const meta = {
...
  'twitter:title': 'Open Collective',
...
}
if (title) {
  meta['twitter:title'] = `${title} - Open Collective`
}
```

Comment (NotJohs) [reply src/components/Header.js]: Alright. will fix.

**Pull request merged**